docs/api/resnet_pretrained_weights.csv

```diff
@@ -2,9 +2,29 @@ Weight,Channels,Source,Citation,BigEarthNet,EuroSAT,So2Sat,OSCD
 ResNet18_Weights.SENTINEL2_ALL_MOCO,13,`link <https://github.com/zhu-xlab/SSL4EO-S12>`__,`link <https://arxiv.org/abs/2211.07044>`__,,,,
 ResNet18_Weights.SENTINEL2_RGB_MOCO, 3,`link <https://github.com/zhu-xlab/SSL4EO-S12>`__,`link <https://arxiv.org/abs/2211.07044>`__,,,,
 ResNet18_Weights.SENTINEL2_RGB_SECO, 3,`link <https://github.com/ServiceNow/seasonal-contrast>`__,`link <https://arxiv.org/abs/2103.16607>`__,87.27,93.14,,46.94
+ResNet18_Weights.LANDSAT_ETM_SR_MOCO,6,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_ETM_SR_SIMCLR,6,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_ETM_TOA_MOCO,9,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_ETM_TOA_SIMCLR,9,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_OLI_SR_MOCO,7,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_OLI_SR_SIMCLR,7,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_OLI_TIRS_TOA_MOCO,11,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_OLI_TIRS_TOA_SIMCLR,11,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_TM_TOA_MOCO,5,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
+ResNet18_Weights.LANDSAT_TM_TOA_SIMCLR,5,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
```

Comment:
Can you sort these in the same order as the table in our paper? The goal is basically to copy the table from our paper so people know which weights to prefer. We also need to clarify that TM = Landsat 4–5 and ETM+ = L7 and OLI/TIRS = L8–9.

Reply:
do you want both accuracy and mIoU reported for NLCD and CDL?

Reply:
do you want both accuracy and mIoU reported for NLCD and CDL?

Reply:
Yeah, let's do that. The table might be too wide but I think it adds an option to scroll. Will have to see what the table looks like in the doc preview.